src/puppetlabs/trapperkeeper/services/authorization/authorization_service.clj

```diff
@@ -19,7 +20,7 @@
    [this context]
    (let [config (get-in-config [:authorization])
          rules (-> config validate-auth-config! :rules transform-config)]
-     (log/debug "Transformed auth.conf rules:\n" (ks/pprint-to-string rules))
+     (log/debug (trs "Transformed auth.conf rules:") "\n" (ks/pprint-to-string rules))
```

Comment:
Yeah, I think it does.  I was just thinking we wanted to always have the full body of the message wrapped inside of the trs format string, not that it would necessarily influence how this particular string might be translated.